### PR TITLE
Bring plugin up-to-date with recent versions of Pandoc

### PR DESCRIPTION
**Pull request by bk**
Harmonize ikiwiki-pandoc with recent versions of pandoc, including:

- Parse YAML meta blocks and correct parsing of old-stylle meta block (title, author, date).
- Improve bibliography/citation support.
- Get rid of deprecated `--html5` parameter when pandoc is called.
- Add support for Org and OPML files.
- Improve math support and handle all math settings using same `page.tmpl` template. Add support for KaTeX.
- Add filter support.
- New options in *.setup file: pandoc_citeproc, pandoc_csl, pandoc_org, pandoc_opml, pandoc_math_custom_js, pandoc_math_custom_css, pandoc_filters.

### CODE REVIEW

File: pandoc.pm [hunk 1]
```diff
@@ -8,6 +8,7 @@ use IkiWiki;
 use FileHandle;
 use IPC::Open2;
 use JSON;
+# use Data::Dumper;
```
Comment (sciunto) [inline pandoc.pm]: Do we need to keep this?

**Comment by sciunto** [comment]
It's neat. Very nice job! I'm going to test it on my server and merge it very soon.

Thank you!

File: pandoc.pm (hunk 1 above)
Comment (bk) [inline pandoc.pm]: No.

**Comment by sciunto** [comment]
I merge this as it works perfectly for me.

Many many thanks! @bk

**Pull request merged**